src/code/views/value-slider-view.tsx

```diff
@@ -240,21 +283,51 @@ export class SVGSliderView extends React.Component<SVGSliderViewProps, SVGSlider
   }
 
   private handleDrag = (e) => {
+    if (this.isButtonEvent(e)) {
+      return;
+    }
     this.handleNoop(e);
+    this.sliderMoved = true;
     const { onValueChange } = this.props;
     if (onValueChange == null) { return; }
 
     const value = this.position(e);
     if (value !== this.props.value) {
+      this.checkForSliderWiggle(value);
       return onValueChange(value);
     }
   }
 
   private handleJumpAndDrag = (e) => {
+    if (this.isButtonEvent(e)) {
+      return;
+    }
     this.handleDrag(e);
     return this.handleStart(e);
   }
 
+  // shows the buttons if the user moves the slider around a small range
+  private checkForSliderWiggle(value: number) {
+    if (!this.state.showButtons) {
+      const {wiggleList} = this;
+      const {length} = wiggleList;
+      wiggleList.push(value);
+      if (length >= SliderWiggleLookback) {
```

Comment:
I might need a description of what we are doing in this loop. I got lost.

1. We are looking at the last `n` changes in absolute model `value` history (`wiggleList`).
2. If the maximum span (between the lowest and highest value) in our list is <= `maxChange` then we show the buttons.

I guess that makes sense.  `maxChange` is a threshold value, if we go over it, we should also `setState({showButtons: false})` I would think?  I am not sure  what to call `maxChange` but the name is a little confusing to me.

Reply:
This is meant to show the buttons when the user is "wiggling" around a value but not hide them if they are already showing.  After playing with it with Dan on Friday its not exactly what he wants.  I'm going to change it today to store deltas and see if that gets closer to the behavior he wants.